livespark-form-modeler/livespark-form-modeler-api/src/main/java/org/livespark/formmodeler/model/impl/basic/SliderFieldDefinition.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.livespark.formmodeler.model.impl.basic;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+import org.jboss.errai.databinding.client.api.Bindable;
+import org.livespark.formmodeler.metaModel.FieldDef;
+import org.livespark.formmodeler.model.FieldDefinition;
+
+/**
+ * @author Pere Fernandez <pefernan@redhat.com>
+ */
+@Portable
+@Bindable
+public class SliderFieldDefinition extends FieldDefinition {
+
+    public static final String CODE = "Slider";
+
+    private String code = CODE;
+
+    @FieldDef( label = "Min. Value" )
+    private Double min = new Double( 0 );
```

Comment:
So it might not matter because this is client-code, but I think it is usually better to use a literal value (i.e. `0.0`) or `Double.valueOf` because it gives the runtime/compiler a chance to cache instances.